README.md

```diff
@@ -180,27 +180,21 @@ Now supposing you want to interact with the device, you have to consume its Thin
 ```JavaScript
 // client.js
 // Required steps to create a servient for a client
-const { Servient, Helpers } = require("@node-wot/core");
+const { Servient } = require("@node-wot/core");
 const { HttpClientFactory } = require('@node-wot/binding-http');
 
 const servient = new Servient();
 servient.addClientFactory(new HttpClientFactory(null));
-const WoTHelpers = new Helpers(servient);
-
-WoTHelpers.fetch("http://localhost:8080/counter").then(async (td) => {
-    try {
-        const WoT = await servient.start();
-        // Then from here on you can consume the thing
-        let thing = await WoT.consume(td);
-        thing.observeProperty("count", async (data) => { console.log("count:", await data.value()); });
-        for (let i = 0; i < 5; i++) {
-            await thing.invokeAction("increment");
-        }
-    }
-    catch (err) {
-        console.error("Script error:", err);
+
+servient.start().then(async (WoT) => {
+    const td = await WoT.requestThingDescription("http://localhost:8080/counter");
+    // Then from here on you can consume the thing
+    let thing = await WoT.consume(td);
+    thing.observeProperty("count", async (data) => { console.log("count:", await data.value()); });
+    for (let i = 0; i < 5; i++) {
+        await thing.invokeAction("increment");
     }
```

Comment:
Question: Should we keep the "internal" try catch ?

Reply:
Hmm, since there is a `.catch` down below, keeping it as it is should be fine, I suppose 🤔 I think we could also consider removing the `"Error:"` prefix, since the type of error should be part of the output if I am not mistaken.